pkg/feature/servicemesh/resources.go

```diff
@@ -31,27 +31,28 @@ func MeshRefs(f *feature.Feature) error {
 
 // AuthRefs stores authorization configuration in the config map, so it can
 // be easily accessed by other components which rely on this information.
-func AuthRefs(f *feature.Feature) error {
-	audiences := f.Spec.Auth.Audiences
-	namespace := f.Spec.AppNamespace
-
-	audiencesList := ""
-	if audiences != nil && len(*audiences) > 0 {
-		audiencesList = strings.Join(*audiences, ",")
-	}
-	data := map[string]string{
-		"AUTH_AUDIENCE":   audiencesList,
-		"AUTH_PROVIDER":   namespace + "-auth-provider",
-		"AUTHORINO_LABEL": "security.opendatahub.io/authorization-group=default",
+func AuthRefs(audiences []string) feature.Action {
+	return func(f *feature.Feature) error {
+		namespace := f.Spec.AppNamespace
+
+		audiencesList := ""
+		if len(audiences) > 0 {
+			audiencesList = strings.Join(audiences, ",")
+		}
+		data := map[string]string{
+			"AUTH_AUDIENCE":   audiencesList,
+			"AUTH_PROVIDER":   namespace + "-auth-provider",
+			"AUTHORINO_LABEL": "security.opendatahub.io/authorization-group=default",
```

Comment:
should we move this label value to `pkg/metadata/labels` ?

Reply:
good idea, done here
c99d6d7